client/view/workbook/sheet/memorize/memorize.js

```diff
@@ -4,4 +4,19 @@ export default mediator => ({
 	name: `workbook.sheet.memorize`,
 	route: `memorize`,
 	template: Memorize,
+	async resolve(data, { key, sheetId }) {
+		const sheet = await mediator.call(`getSheet`, key, sheetId)
```

Comment:
Did you check to see if it makes two HTTP requests when two `getSheet` calls are made at once?  I'm not actually sure if that would happen in Chrome/Firefox/Safari or not

Reply:
Chrome and Firefox each make three requests, Safari makes two.